allennlp/training/trainer.py

```diff
@@ -125,18 +143,16 @@ def __init__(self,
         self._log_interval = 10  # seconds
         self._summary_interval = 100  # num batches between logging to tensorboard
 
-    def train(self) -> None:
-        epoch_counter = 0
-        # Resume from serialization path if it contains a saved model.
-        if self._serialization_dir is not None:
-            # Set up tensorboard logging.
-            train_log = SummaryWriter(os.path.join(self._serialization_dir, "log", "train"))
-            validation_log = SummaryWriter(os.path.join(self._serialization_dir, "log", "validation"))
-            if any(["model_state_epoch_" in x
-                    for x in os.listdir(self._serialization_dir)]):
-                logger.info("Loading model from checkpoint.")
-                epoch_counter = self._restore_checkpoint()
+        self._last_log = 0.0  # time of last logging
 
+        if serialization_dir is not None:
+            train_log = SummaryWriter(os.path.join(serialization_dir, "log", "train"))
+            validation_log = SummaryWriter(os.path.join(serialization_dir, "log", "validation"))
+            self._writer = TensorboardWriter(train_log, validation_log)
+        else:
+            self._writer = TensorboardWriter()
+
+    def _enable_gradient_clipping(self) -> None:
         if self._grad_clipping is not None:
             # Pylint is unable to tell that we're in the case that _glad_clipping is not None...
```

Comment:
Just noticed this comment says "_glad_clipping" instead of "_grad_clipping"...  Care to fix my typo?

Reply:
done